authservice/src/main/java/auth/proximity/authservice/security/jwt/JwtConstants.java

```diff
@@ -2,9 +2,9 @@
 
 public class JwtConstants {
 
-    public static final long EXPIRE_ACCESS_TOKEN = 30 * 60 * 1000; //5 * 60 * 1000 = 30 minutes
```

Comment:
This can also change depending on the environment so  it shouldn't be hard coded. use properties